exasol_integration_test_docker_environment/lib/test_environment/spawn_test_database.py

```diff
@@ -236,13 +237,15 @@ def _add_exa_conf(self, copy: DockerContainerCopy,
             PACKAGE_NAME,
             f"{self.docker_db_config_resource_name}/EXAConf") # type: bytes
         template = Template(template_str.decode("utf-8"))
+        additional_db_parameter_str = " ".join([f"-{p}" for p in self.additional_db_parameter])
```

Comment:
This is a hard assumption, that everyone cli argument start with a single minus

Reply:
If i change the cmd line interface I`am afraid readibilty gets worse:
```
--additional-db-parameter disableIndexIteratorScan=1
```
is better understandable as
```
--additional-db-parameter -disableIndexIteratorScan=1
```
, essentially if there are multiple additional db parameters.

Reply:
Hm, I have changed the format